### PR TITLE
Feat/elements unit testing

### PR DESCRIPTION
**Pull request by Dandy24**

### CODE REVIEW

File: empmanager/frontend2/src/components/form/elements/__test__/MainNavigation.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        </BrowserRouter>,
+    );
+
+    const CompanyListEl = getByTestId('menu-company-list-item');
+
+    expect(CompanyListEl.textContent).toBe('Seznam firem');
+});
+
+test('Menu items link correctly redirects to right page', () => {
+    Object.defineProperty(window, 'matchMedia', {
```
Comment (Kobzol) [inline empmanager/frontend2/src/components/form/elements/__test__/MainNavigation.test.js:35]: Why is this needed?

Comment (Dandy24) [reply empmanager/frontend2/src/components/form/elements/__test__/MainNavigation.test.js:35]: Jest will crash without defining this property. According to their own documentation, they have not yet completed 
their implemented of a method Window.matchMedia() which is used in this test.
 https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom

I found this property definition to be their official solution, according to multiple threads.

Comment (Kobzol) [reply empmanager/frontend2/src/components/form/elements/__test__/MainNavigation.test.js:35]: Ok :) In such "non-obvious" cases, I suggest to include a comment with a link that explains what is going on.

Comment (Dandy24) [reply empmanager/frontend2/src/components/form/elements/__test__/MainNavigation.test.js]: Good idea, added explanation in comment
